doc/zh-cn/admin-api.md

```diff
@@ -528,6 +534,8 @@ APISIX 的 Upstream 除了基本的复杂均衡算法选择外，还支持对上
 |pass_host            |可选|枚举|`pass` 透传客户端请求的 host, `node` 不透传客户端请求的 host, 使用 upstream node 配置的 host, `rewrite` 使用 `upstream_host` 配置的值重写 host 。||
 |upstream_host    |可选|辅助|只在 `pass_host` 配置为 `rewrite` 时有效。||
 |labels   |可选 |匹配规则|标识附加属性的键值对|{"version":"v2","build":"16","env":"production"}|
+|create_time|可选|辅助|单位为秒的 epoch 时间戳|1602883670|
```

Comment:
Should we translate the `epoch` word?

![image](https://user-images.githubusercontent.com/2106987/98954379-b0d3d100-2538-11eb-8867-5267e2cd9c98.png)